File: src/test/java/br/com/newObjects/model/entities/classes/NovoPedido.java
package br.com.newObjects.model.entities.classes;

import java.util.Random;

import br.com.model.entities.classes.FormaPagamento;
import br.com.model.entities.classes.Pedido;
import br.com.newObjects.model.entities.classes.usuario.NovoCliente;

public class NovoPedido {

    public static Pedido novoPedido() {

        Pedido pedido = new Pedido(NovoCliente.novoCliente(),
                FormaPagamento.novaFormaPagamento(new Random().nextInt(5) + 1),
                new Random().nextFloat(10));

        NovoItemPedido.novaListaItensPedido(pedido,
                NovoProdutoFornecedor.novaListaProdutosFornecedor());

        NovaAvaliacao.novaAvaliacao(pedido);

        return pedido;
    }

    public static Pedido novoPedidoSimples() {
        return new Pedido(NovoCliente.novoCliente(), FormaPagamento.novaFormaPagamento(new Random().nextInt(5) + 1),
                new Random().nextFloat(100) + 1);
    }
}

File: src/test/java/br/com/newObjects/model/NovoPedido.kt
package br.com.newObjects.model

import br.com.model.FormaPagamento.Companion.novaFormaPagamento
import br.com.model.Pedido
import br.com.newObjects.model.usuario.NovoCliente
import java.util.*

object NovoPedido {
    fun novoPedido(): Pedido {
        val pedido = Pedido(
            cliente = NovoCliente.novoCliente(),
            formaPagamento = novaFormaPagamento(Random().nextInt(5) + 1),
            desconto = Random().nextFloat(10f)
        )
        NovoItemPedido.novaListaItensPedido(
            pedido, NovoProdutoFornecedor.novaListaProdutosFornecedor()
        )
        NovaAvaliacao.novaAvaliacao(pedido)
        return pedido
    }

    fun novoPedidoSimples(): Pedido {
        return Pedido(
            cliente = NovoCliente.novoCliente(),
            formaPagamento = novaFormaPagamento(Random().nextInt(5) + 1),
            desconto = Random().nextFloat(100f) + 1
        )
    }
}
